enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -70,6 +71,8 @@ Some features of cgroup v2 include:
 
 - [ ] Enable cgroup v2 within the Openshift API
 - [ ] Add kernel flags to MCO to enable cgroup v2 on nodes
+- [ ] Remove support to configure cgroup v1 in OCP clusters
```

Comment:
it's a little sparse on exactly how this will happen. I personally think an admission webhook would be best to prevent updates from going through

Reply:
Makes sense. Updated the goal accordingly